projects/js-packages/components/components/action-popover/styles.module.scss

```
@@ -0,0 +1,49 @@
+.wrapper {
+	width: 304px;
+	min-height: 230px;
```

could we define these values based on the `--spacing-base` too?

```suggestion
	width: calc( var( --spacing-base ) * 38 ); // 304px
	min-height: calc( var( --spacing-base ) * 29 ); // almost 302px!;
```

---

hmm! don't think we should use `spacing` with `sizes`, these are different things when we talk about design, and if in some moment we decide to update the spacing value we shouldn't affect the sizes.

---

yeah, you're right. What I'd like to guarantee is defining the sizes also based on  multiples of eight

---

agree that we should use multiples, it sounds better on readability than random numbers, but maybe we could introduce a new variable or do it manually?